Lib/copyreg.py

```diff
@@ -36,6 +36,12 @@ def pickle_complex(c):
 
     pickle(complex, pickle_complex, complex)
 
+def pickle_union(obj):
```

Comment:
Off topic: I didn't know such a module existed 😮 . This is really cool! Thanks for teaching me something new today Serhiy.